### PR TITLE
Improved contributing.md file 

### LINKED ISSUE #921: Update documentation throughout repo
Replacement of https://github.com/hotosm/fmtm/issues/164

We currently have some minimal documentation, which could be greatly improved.
We would really welcome PRs for this 🙏 

## What needs to be done:

- Proof reading of current docs and correction.
- Reduce docs duplication - we currently have the same content duplicated in a few places.
- Improve general README, contributors guidelines, etc.
- Improve developer documentation with additional examples, FAQs (if you struggled setting up FMTM anywhere, feel free to document it!).
- Improve user documentation for how to use FMTM.
- Python docstrings throughout the backend, where missing (especially on routes).
- If you particularly like writing JSDoc docstrings, then it wouldn't harm having a few in the frontend too.

### PR DESCRIPTION
**Pull request by vinayakhegde1**
This PR fixes #921 
Updated the contributing guidelines made it more simple corrected some words

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
 ## 🤗 Welcome
```
Comment (spwoodcock) [inline CONTRIBUTING.md:1]: Could you make this a top level heading please? 🙏 `#`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 
-:+1::tada: First off, We are really glad you're reading this, because we need volunteer developers to help improve the Field Mapping Tasking Manager (FMTM)! :tada::+1:
-
-We welcome and encourage contributors of all skill levels, and we are committed to making sure your participation is inclusive, enjoyable, and rewarding. If you have never contributed to an open source project before, we are a good place to start, and we will make sure you are supported every step of the way. If you have **any** questions, please ask!
+👍🎉 First off, We are really glad you're reading this, because we need volunteer developers to help improve the Field Mapping Tasking Manager (FMTM)! Welcome to Field Mapping Tasking Manager (FMTM)! We're excited to have you join our community of contributors. Whether you're new to open source or an experienced developer, we value your contributions and are here to support you every step of the way. If you have any questions or need assistance, please don't hesitate to ask. 🎉👍
```
Comment (spwoodcock) [inline CONTRIBUTING.md:3]: I actually preferred the previous wording, so sorry 😅 
It's nice to state that `participation is inclusive, enjoyable, and rewarding`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,22 +1,20 @@
...
 
 You can see an overview of the project and the process we have gone through in developing FMTM so far in these [slides](https://docs.google.com/presentation/d/1UrBG1X4MXwVd8Ps498FDlAYvesIailjjPPJfR_B4SUs/edit#slide=id.g15c1f409958_0_0) .
 
-Furthermore, there are many ways to contribute to the **Field Mapping Tasking Manager (FMTM)**, which includes:
+How to Contribute
```
Comment (spwoodcock) [inline CONTRIBUTING.md:7]: Please make this a level 2 heading `##`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
+
+There are several ways to contribute to the Field Mapping Tasking Manager:
 
 ## Testing
```
Comment (spwoodcock) [inline CONTRIBUTING.md:11]: As these headings go below `How to contribute` now, they can be level 3 headings `###`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,22 +1,20 @@
...
 
-Right now, we are in the process of building the prototype. We warmly welcome your input in testing and sharing your feedback. If you are also interested in coordinating a field testing session, please reach out!
+We're currently building the prototype, and your input is invaluable. Please help us by testing and providing feedback. If you're interested in coordinating field testing sessions, please reach out.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:13]: We currently have an alpha release, so any testing and feedback would be warmly welcomed. If you're interested in coordinating field testing sessions, please reach out.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
 
 ## Report bugs and suggest improvements
 
 The [issue queue](https://github.com/hotosm/fmtm/issues) is the best way to get started. There are issue templates for BUGs and FEATURES that you can use, you could also create your own. Once you have submitted an issue, it will be assigned one label from the following [label categories](https://github.com/hotosm/fmtm/labels). If you are wondering where to start, you can filter by the **good first issue label**.
 
 ## :handshake: Thank you
 
-Thank you very much in advance for your contributions!! Please ensure you refer to our **Code of Conduct**.
-If you've read the guidelines, but are still not sure how to contribute on Github, please reach out to us via our Slack **#geospatial-tech-and-innovation**.
+We're grateful for your contributions. Please review our **Code of Conduct**.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:34]: Perhaps update this to: `We are grateful for your contributions`. Then move `Please review our **Code of Conduct**.`  into the `## Code Contributions` section.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
+If you have any questions about contributing on GitHub, don't hesitate to reach out to us via our Slack channel, **#geospatial-tech-and-innovation**.
 
 # Code Contribution guidelines
 
 ## Workflow
 
-We operate the "Fork & Pull" model explained at [About Pull
-Requests](https://help.github.com/articles/about-pull-requests/)
+We follow the "Fork & Pull" model for contributions it is explained at [About Pull Requests](https://help.github.com/articles/about-pull-requests/). Here's how it works:
```
Comment (spwoodcock) [inline CONTRIBUTING.md:41]: `We follow the [Fork & Pull](https://help.github.com/articles/about-pull-requests/) model for contributions:`

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
+
+Fork the Project: Fork the project into your own repository.
+
+Create a Topic Branch: Create a topic branch in your repository to work on your contribution.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:45]: `Create a Feature Branch`

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
 
 # Code Contribution guidelines
 
 ## Workflow
 
-We operate the "Fork & Pull" model explained at [About Pull
-Requests](https://help.github.com/articles/about-pull-requests/)
+We follow the "Fork & Pull" model for contributions it is explained at [About Pull Requests](https://help.github.com/articles/about-pull-requests/). Here's how it works:
+
+Fork the Project: Fork the project into your own repository.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:43]: Please add these to either a numbered list, or bullet points.

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
+
+Create a Topic Branch: Create a topic branch in your repository to work on your contribution.
+
+Make Pull Requests: Create one or more pull requests back to the main repository.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:47]: Add the sentence: `Complete PRs are preferred, but Draft PRs are also accepted if the feature/update is large.`

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -25,58 +23,56 @@ Skills with the following would be beneficial:
...
+- Please open a separate issue for each problem, question, or comment you have. Do not re-use existing issues for other topics, even if they are similar. This keeps issues small and manageable and makes it much easier to follow through and make sure each problem is taken care of.
 
 ## Documentation
 
 Project documentation should be in [Markdown
 format](https://www.markdownguide.org/), and in a _docs_
-subdirectory. While it is possible to use HTML in Markdown documents
-for tables and images, it is prefered to use the Markdown style as
-it's much easier to read.
+subdirectory.We prefer using Markdown over HTML in Markdown documents for better readability.
```
Comment (spwoodcock) [inline CONTRIBUTING.md:75]: space between sentences

**Review by spwoodcock — COMMENTED** [review]
Thanks @vinayakhegde1 ! Some good edits here - sorry I left a few comments for update - it would be great if you could address them & I will merge 🙏

**Comment by vinayakhegde1** [comment]
@spwoodcock Thanks for the feedback I'll make the changes

**Comment by spwoodcock** [comment]
Hello @vinayakhegde1 - thank you for your contribution 🙏

The updated changes would be nice to incorporate.

Feel free to make another PR with updates if you would like to.

**Comment by vinayakhegde1** [comment]
Sorry for the inconvenience. I will do it today

**Pull request closed without merging**